proto/api/v1/discovered_cluster_service.proto

```diff
@@ -0,0 +1,135 @@
+syntax = "proto3";
+
+package v1;
+
+import "api/v1/pagination.proto";
+import weak "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "v1";
+option java_package = "io.stackrox.proto.api.v1";
+
+// DiscoveredCluster is the service representation of discovered clusters.
+//
+// A discovered cluster originates from a cloud source integration. If
+// an equivalent internal cluster exists, the discovered cluster is considered
+// secured. If equivalent secured cluster exists, the status of the discovered
+// cluster is either unsecured or unknown, depending on the provider details.
```

Comment:
Should be "not exists"?

Also would be great to have this section as documentation on the status enum.